docs/how-to-guides/sensor-calibration.md

```diff
@@ -0,0 +1,18 @@
+# How to calibrate your sensors
+
+## Overview
+
+Autoware expects to have multiple sensors attached to the vehicle as input to perception, localization, and planning stack. These sensors must be calibrated correctly and the position must be defined as urdf (or as tf launch files) as in [sample_sensor_kit](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main/sample_sensor_kit_description).
```

Comment:
Slightly reordered sentence, since current wording implies that the sample_sensor_kit uses tf launch files rather than urdf files.

```suggestion
Autoware expects to have multiple sensors attached to the vehicle as input to perception, localization, and planning stack. These sensors must be calibrated correctly and their positions must be defined using either urdf files (as in [sample_sensor_kit](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main/sample_sensor_kit_description)) or as tf launch files.
```